trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    nameable.correct_name.length > 10 ? nameable.correct_name[0..10] : nameable.correct_name
```

Comment:
- [ ] Kindly use the instance variable `@nameable` instead of just `nameable` so that you can access the `@nameable` instance variable from the parent class. 
- [ ] Your Trimmer returns `11` characters instead of `10` when passed a name with more than `10` characters. Kindly fix it.
Your line `5` should look like this ` @nameable.correct_name.length > 10 ? @nameable.correct_name[0...10] : @nameable.`